src/arns.lua

```diff
@@ -296,27 +305,24 @@ function arns.assertValidBuyRecord(name, years, purchaseType, processId)
 	end
 
 	-- assert processId is valid pattern
-	assert(type(processId) == "string", "ProcessId is required and must be a string.")
-	assert(utils.isValidBase64Url(processId), "ProcessId pattern is invalid.")
+	assert(type(processId) == "string", "Process id is required and must be a string.")
+	assert(utils.isValidAOAddress(processId), "Process id must be a valid base64url.")
 end
 
+--- Asserts that a record is valid for extending the lease
+--- @param record table|nil The record to check
+--- @param currentTimestamp number|nil The current timestamp
```

Comment:
Why express that any of these can be nil when nil will cause an error to be thrown?